examples/cluster.yml

```diff
@@ -10,10 +10,35 @@ spec:
     customer:
       id: "example-customer"
 
+    docker:
+      imageNamespace: "giantswarm"
+
+    etcd:
+      domain: "k8s.cluster.giantswarm.io"
+      prefix: "example-cluster"
+
+    calico:
+      subnet: "192.168.0.0"
+      cidr: "24"
+      mtu: "1500"
+
+    kubernetes:
+      api:
+        domain: "k8s.cluster.giantswarm.io"
+        insecurePort: "8080"
```

Comment:
I get json errors here, I think it needs to be just a plain int, as well as the line below